srcpkgs/font-Siji/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'font-Siji'
+pkgname=font-Siji
+version=0.0.180725
+revision=1
+build_style=fetch
+hostmakedepends="git"
```

Comment:
remove hostmakedepends= when switching to the fetching commit tarball method described below and add

depends="font-util"